src/views/Entries/QualityControl/EntryCard/index.tsx

```diff
@@ -287,91 +297,113 @@ function EntryCard(props: EntryCardProps) {
+                                >
+                                    { entry.createdByName }
+                                </div>
+                                <DateOutput
+                                    className={styles.createdAt}
+                                    value={entry.createdAt}
+                                    tooltip={_ts('entries.qualityControl', 'entryCreatedOnTooltip')}
                                 />
-                            )}
-                            <div
-                                className={styles.value}
-                                title={_ts('entries.qualityControl', 'leadSourceTooltip', { leadSource })}
-                            >
-                                { lead.sourceDetail ? lead.sourceDetail.title : lead.sourceRaw }
                             </div>
-                        </div>
-                        <div className={styles.confidentiality}>
-                            { lead.confidentialityDisplay }
-                        </div>
-                    </div>
-                    <div className={styles.entryDetailsRow}>
-                        <div
-                            className={styles.createdBy}
-                            title={_ts('entries.qualityControl', 'leadCreatedByTooltip', { user: entry.createdByName })}
+                            <ListView
+                                className={styles.scaleWidgets}
+                                data={scaleWidgets}
+                                renderer={ListItem}
+                                rendererParams={scaleWidgetRendererParams}
+                                keySelector={widgetKeySelector}
+                                emptyComponent={EmptyComponent}
+                            />
+                        </section>
+                    )}
+                    {!isVisible ? (
+                        <Button
+                            className={styles.expandButton}
+                            onClick={handleShow}
+                            transparent
+                            iconName="chevronUp"
                         >
-                            { entry.createdByName }
-                        </div>
-                        <DateOutput
-                            className={styles.createdAt}
-                            value={entry.createdAt}
-                            tooltip={_ts('entries.qualityControl', 'entryCreatedOnTooltip')}
+                            {_ts('entries', 'more')}
+                        </Button>
+                    ) : (
+                        <Button
+                            className={styles.collapseButton}
+                            onClick={handleHide}
+                            transparent
+                            iconName="chevronDown"
+                        >
+                            {_ts('entries', 'less')}
+                        </Button>
```

Comment:
```suggestion
                        <Button
                            className={styles.collapseButton}
                            onClick={isVisible? handleHide : handleShow}
                            transparent
                            iconName={isVisible ? "chevronDown" : "chevronUp"}
                        >
                            {isVisible? _ts('entries', 'less') : _ts('entries', 'more')}
                        </Button>
```

Reply:
If you do this, you can go ahead and remove 2 buttons and use the same one.
Also, it will be more readable.